### PR TITLE
first commit that uses the BRC addiions

### PR DESCRIPTION
**Pull request by scottcain**

### CODE REVIEW

**Comment by scottcain** [comment]
Well, shoot. It looks like I didn't do something right.
```
Exception: Missing environment variables: BRC_MASTODON_ACCESS_TOKEN for mastodon-brc plugin.
```
@wm75 Could you take a look to see if there is anything obvious I did wrong? Thanks!

**Comment by arash77** [comment]
This occurred because the config wasn't applied to the workflow files. It will be resolved by: #128.

**Comment by scottcain** [comment]
Excellent, Thanks @arash77

File: posts/2024/2024-10-10-brc-test.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+media:
+  - mastodon-brc
+  - bluesky-brc
+  - linkedin-brc
+
+mentions:
+  mastodon-eu-freiburg:
+    - brc_analytics@mstdn.science
+  bluesky-galaxyproject:
+    - brcanaltyics.bsky.social
```
[the remark this answers is not in the document]
Comment (scottcain) [reply posts/2024/2024-10-10-brc-test.md]: Thanks for clarifying; I moved the mentions in the other directions, to mention the Galaxy project and Freiburg.

**Review by bgruening — APPROVED** [review]
Good to go from my side. Thanks for the flowers.

**Comment by scottcain** [comment]
@bgruening any need to be on bluesky? No, I'm not a fan.

**Pull request merged**